apps/guardian/src/shared/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import { Spinner } from 'react-bootstrap';
+import RBButton, { ButtonProps } from 'react-bootstrap/Button';
+import { clsx } from 'clsx';
+
+type Props = ButtonProps & {
+    /** Whether the button should show as loading */
+    loading?: boolean;
```

Comment:
It's optional because the convention is to make it possible to declare boolean props on components as `<Button loading />` and `<Button />` instead of explicitly `<Button loading={true} />` and `<Button loading={false} />`.